torch/_meta_registrations.py

```diff
@@ -1867,11 +1899,6 @@ def activate_meta():
             # Instead, we should be letting those decompositions run, and writing meta kernels
             # only for the base operators.
             pass
-        elif op_overload.is_view:
-            # Attempting to register a python meta kernel for a view operator.
-            # We shouldn't do this, because the output will report as not having aliased storages.
-            # All view ops have meta kernels in C++ today, so we should use those instead.
-            pass
```

Comment:
How is meta kernel working now for view ops?

Reply:
I fixed the meta on prim.as_strided, so we should be indifferent to views being done in python or c++